src/app/api/proxy/[...proxy]/route.ts

```diff
@@ -110,13 +95,6 @@ export async function POST(req: NextRequest) {
 
     // Proxy the request
     const proxyUrl = `${BASE_PROXY_URL}${path}`;
-    const response = await fetch(proxyUrl, {
-      method: "POST",
-      headers: {
-        "Content-Type": "application/json",
-      },
-      body, // Forward the raw body
-    });
```

Comment:
Just checking here: we did not use `response`, but we were making the call. If it had side effects, we might not be triggering them anymore. Is that intentional?

Reply:
So previously we were sending the same request to both mempool api and bitcoind?

Reply:
yes basically this was calling the mempool API's post tx and it always fails the line after it does it through the bitcoin rpc directly to post the tx